examples/openshift-compatible/README.md

```diff
@@ -45,8 +46,25 @@ prometheus-node-exporter:
 grafana-agent:
   agent:
     listenPort: 8080
+    securityContext:
```

Comment:
Is there an argument against applying this securityContext to all platforms? I'm all for safer defaults :wink:

Reply:
it's a good discussion that we should have with the Agent team.